stdlib/runtime/malloc.gr

```diff
@@ -174,7 +172,8 @@ export let free = (ap: WasmI32) => {
     // Find the location to insert this block into the free list
     while (true) {
       let nextp = getNext(p)
-      if ( ((blockPtr > p) && (blockPtr < nextp)) || ((p >= nextp) && ((blockPtr > p) || (blockPtr < nextp))) ) {
+      if (blockPtr > p && blockPtr < nextp ||
+      p >= nextp && (blockPtr > p || blockPtr < nextp)) {
```

Comment:
Should probably be indented

Reply:
Agreed.  Indents.  Let me know if it should reallllly indent and break into
(
  cond &&
  cond
)

Reply:
Fixed.